Makefile

```diff
@@ -138,20 +138,20 @@ $(FFMPEG_ROOT):
 	git -C $(FFMPEG_ROOT) checkout release/6.1
 
 # For ARM64 builds, use the image ghcr.io/viamrobotics/antique:arm64 for backward compatibility
-FFMPEG_H264_PREFIX ?= $(shell realpath .)/gostream/codec/h264/ffmpeg/$(shell uname -s)-$(shell uname -m)
-ffmpeg-h264-static: $(FFMPEG_ROOT)
-	cd $(FFMPEG_ROOT) && ./configure \
-		--disable-programs \
-		--disable-doc \
-		--disable-everything \
-		--enable-encoder=h264_v4l2m2m \
-		--prefix=$(FFMPEG_H264_PREFIX) \
-		--enable-pic
+FFMPEG_PREFIX ?= $(shell realpath .)/gostream/ffmpeg/$(shell uname -s)-$(shell uname -m)
+FFMPEG_OPTS = --disable-programs --disable-doc --disable-everything --prefix=$(FFMPEG_PREFIX) --enable-pic --disable-autodetect
+ifeq ($(shell uname -m),aarch64)
+	FFMPEG_OPTS += --enable-encoder=h264_v4l2m2m
+	FFMPEG_OPTS += --enable-v4l2-m2m
+endif
+ffmpeg: $(FFMPEG_ROOT)
```

Comment:
unrelated -- if FFMPEG_ROOT step takes a long time, you can change it to
```
git clone https://github.com/FFmpeg/FFmpeg.git --depth 1 --branch release/6.1
```

this will fetch 20mb instead of hundreds

Reply:
Nice. Done!